### PR TITLE
UX Redesign: Calculator intro, location section, and activity section

### PR DESCRIPTION
**Pull request by blanchardjeremy**

### CODE REVIEW

File: src/components/calculator/PointsDisplay.tsx [hunk 1]
```diff
@@ -46,6 +46,40 @@ export function ExplanationCard(props: {
             : ''}
         </b>
       </p>
+
```
Comment (beshaya) [inline src/components/calculator/PointsDisplay.tsx:49]: Please remove dead code.

File: src/pages/Calculator.tsx [hunk 2]
```diff
@@ -186,60 +179,63 @@ export const Calculator = (): React.ReactElement => {
...
-                  />
-
-                  <SelectControl
+                  <GenericSelectControl
                     id="interaction"
                     label="Is this a single activity or an ongoing relationship?"
-                    data={calculatorData}
-                    setter={setCalculatorData}
+                    setter={(value) =>
+                      setCalculatorData({
```
Comment (beshaya) [inline src/pages/Calculator.tsx:194]: Add a comment here for why this is special.

**Review by beshaya — APPROVED** [review]
Left two cleanup requests, otherwise looks good. Also, this is changing a lot more than just the one-column layout. Please update the PR title to reflect this.

This would have been easier to review as 2-3 PR's.

File: src/components/calculator/PointsDisplay.tsx (hunk 1 above)
Comment (blanchardjeremy) [reply src/components/calculator/PointsDisplay.tsx:49]: It's removed in #346. And I'm happy to remove it here too for consistency.

File: src/pages/Calculator.tsx (hunk 2 above)
Comment (blanchardjeremy) [reply src/pages/Calculator.tsx:194]: Done

**Pull request merged**